ambry-commons/src/test/java/com.github.ambry.commons/BlobIdTest.java

```diff
@@ -160,14 +160,19 @@ public void testComparisons() throws Exception {
         BlobId blobIdV1 = getRandomBlobId(BLOB_ID_V1);
         BlobId blobIdV2 = getRandomBlobId(BLOB_ID_V2);
         BlobId blobIdV3 = getRandomBlobId(BLOB_ID_V3);
+        BlobId blobIdV4 = getRandomBlobId(BLOB_ID_V4);
 
         assertTrue("blobIdV1 should be less than blobIdV2", blobIdV1.compareTo(blobIdV2) < 0);
         assertFalse(blobIdV1.equals(blobIdV2));
         assertTrue("blobIdV1 should be less than blobIdV3", blobIdV1.compareTo(blobIdV3) < 0);
         assertFalse(blobIdV1.equals(blobIdV3));
         assertTrue("blobIdV2 should be less than blobIdV3", blobIdV2.compareTo(blobIdV3) < 0);
         assertFalse(blobIdV2.equals(blobIdV3));
+        assertTrue("blobIdV3 should be less than blobIdV4", blobIdV3.compareTo(blobIdV4) < 0);
```

Comment:
for V3 and above, the comparison is to be done via the uuid only, so V3 < V4 is not always true.